JASP-Desktop/components/JASP/Controls/ComboBox.qml

```diff
@@ -16,8 +16,10 @@ JASPControl
 	property alias	control:				control
 	property alias	controlLabel:			controlLabel
 	property alias	label:					controlLabel.text
-	property string	currentText				//Am i empty or what?
```

Comment:
Would've been nice if you added an explanation here, because Im pretty sure I added those comments. Because I could not figure out (fast enough) what was the purpose of these properties and where/how they were filled/used.

I would really appreciate if you do add some explanation there.

Reply:
Because, currentText means currentLabel? Or what?